youtube-dl/Dockerfile

```diff
@@ -5,18 +5,9 @@ FROM --platform=${TARGETPLATFORM:-linux/amd64} alpine:3.21.3 as ship
 COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
 RUN chmod +x /usr/bin/fwatchdog
 
-RUN apk add --no-cache curl bash ca-certificates
-
-# youtube-dl appears to be broken
-# https://github.com/ytdl-org/youtube-dl/issues/31530
-# Switching to yt-dlp
-# RUN pip install --upgrade yt-dlp
-
-RUN curl -SLs https://get.arkade.dev | sh && \
-    arkade get yt-dlp && \
-    mv $HOME/.arkade/bin/yt-dlp /usr/local/bin/ && \
-    chmod +x /usr/local/bin/yt-dlp && \
-    rm -rf /tmp/* && rm -rf /usr/local/bin/arkade
+# Download yt-dlp, an updated alternative to youtube-dl, from Alpine repository
+# since it is packaged as static binary compatible with Alpine.
+RUN apk add --no-cache ca-certificates yt-dlp
```

Comment:
I read that yt-dlp required Node or Bun etc now going forward to solve captchas.

Thoughts?

Did you want to also update the alpine version.